homeassistant/components/touchline/climate.py

```diff
@@ -1,21 +1,35 @@
-"""Platform for Roth Touchline heat pump controller."""
+"""Platform for Roth Touchline floor heating controller."""
 import logging
-from typing import List
+from collections import OrderedDict
 
 from pytouchline import PyTouchline
 import voluptuous as vol
 
 from homeassistant.components.climate import PLATFORM_SCHEMA, ClimateDevice
 from homeassistant.components.climate.const import (
     HVAC_MODE_HEAT,
+    SUPPORT_PRESET_MODE,
     SUPPORT_TARGET_TEMPERATURE,
 )
 from homeassistant.const import ATTR_TEMPERATURE, CONF_HOST, TEMP_CELSIUS
 import homeassistant.helpers.config_validation as cv
 
 _LOGGER = logging.getLogger(__name__)
 
-SUPPORT_FLAGS = SUPPORT_TARGET_TEMPERATURE
+PRESET_MODES1 = OrderedDict()
```

Comment:
We don't have to use OrderedDict anymore, as the order of dictionaries is guaranteed in the versions of Python Home Assistant uses.

Reply:
Ok, I'll remove the OrderedDict definition.